### PR TITLE
[v3.8.6] Optimize code size for webgl-texture.ts/webgl2-texture.ts

### PR DESCRIPTION
**Pull request by dumganhar**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/gfx/webgl/webgl-texture.ts [hunk 1]
```diff
@@ -94,30 +98,32 @@ export class WebGLTexture extends Texture {
...
             }
 
-            this._viewInfo.texture = this;
-            this._viewInfo.type = info.type;
-            this._viewInfo.format = info.format;
-            this._viewInfo.baseLevel = 0;
-            this._viewInfo.levelCount = info.levelCount;
-            this._viewInfo.baseLayer = 0;
-            this._viewInfo.layerCount = info.layerCount;
+            thisViewInfo.texture = self;
```
Comment (dumganhar) [inline cocos/gfx/webgl/webgl-texture.ts:106]: Adding varible `thisViewInfo` to avoid `this._viewInfo` being inovked many times.

**Comment by dumganhar** [comment]
@cocos-robot run test cases

**Review by star-e — APPROVED** [review]

File: cocos/gfx/webgl/webgl-texture.ts [hunk 2]
```diff
@@ -45,39 +45,43 @@ export class WebGLTexture extends Texture {
...
-            this._info.format,
-            this.width,
-            this.height,
-            this.depth,
-            this._info.levelCount,
-        ) * this._info.layerCount;
-
-        if (!this._isTextureView) {
-            this._gpuTexture = {
+        const thisTextureInfo = this._info;
```
Comment (minggo) [inline cocos/gfx/webgl/webgl-texture.ts]: this -> self

Comment (dumganhar) [reply cocos/gfx/webgl/webgl-texture.ts]: Good catch.

Comment (dumganhar) [reply cocos/gfx/webgl/webgl-texture.ts]: Updated.

**Review by minggo — APPROVED** [review]

**Pull request merged**